config.py

```diff
@@ -5,18 +5,15 @@
 from watchdog.events import FileSystemEventHandler
 from watchdog.observers.polling import PollingObserver as Observer
 
-from consts import RESET_COLOR
-
-log_format = "%(asctime)s %(levelname)s: %(message)s{}".format(RESET_COLOR)
-logging.basicConfig(format=log_format)
-logger = logging.getLogger(__name__)
+from consts import logger
```

Comment:
Can't do this... `__name__` will always be "consts" and we would like to know which module produced the log

Reply:
What I should do? We probably don't want create in every module new logger object and load config to set correct level (#19)

Reply:
Each model should have it's own logger object, that's the common way

Reply:
So I should pass `log_level` to every class to set correct log level?